packages/at_commons/lib/src/verb/enroll_verb_builder.dart

```diff
@@ -35,34 +36,57 @@ class EnrollVerbBuilder extends AbstractVerbBuilder {
   String? encryptedDefaultSelfEncryptionKey;
   String? encryptedAPKAMSymmetricKey;
 
+  /// Filters enrollment requests according to the provided [EnrollmentStatus] criteria.
+  ///
+  /// Accepts a list of enrollment statuses, defaulting to all available status values.
+  List<EnrollmentStatus> enrollmentStatusFilter = EnrollmentStatus.values;
+
   @override
   String buildCommand() {
     var sb = StringBuffer();
     sb.write('enroll:');
     sb.write(getEnrollOperation(operation));
 
-    Map<String, dynamic> enrollParams = (EnrollParams()
-          ..enrollmentId = enrollmentId
-          ..appName = appName
-          ..deviceName = deviceName
-          ..apkamPublicKey = apkamPublicKey
-          ..otp = otp
-          ..namespaces = namespaces
-          ..encryptedDefaultEncryptionPrivateKey =
-              encryptedDefaultEncryptionPrivateKey
-          ..encryptedDefaultSelfEncryptionKey =
-              encryptedDefaultSelfEncryptionKey
-          ..encryptedAPKAMSymmetricKey = encryptedAPKAMSymmetricKey)
-        .toJson();
-    enrollParams
-        .removeWhere((key, value) => value == null || value.toString().isEmpty);
-    if (enrollParams.isNotEmpty) {
-      sb.write(':${jsonEncode(enrollParams)}');
+    EnrollParams enrollParams = EnrollParams()
+      ..enrollmentId = enrollmentId
+      ..appName = appName
+      ..deviceName = deviceName
+      ..apkamPublicKey = apkamPublicKey
+      ..otp = otp
+      ..namespaces = namespaces
+      ..encryptedDefaultEncryptionPrivateKey =
+          encryptedDefaultEncryptionPrivateKey
+      ..encryptedDefaultSelfEncryptionKey = encryptedDefaultSelfEncryptionKey
+      ..encryptedAPKAMSymmetricKey = encryptedAPKAMSymmetricKey
+      ..enrollmentStatusFilter = enrollmentStatusFilter;
+
+    Map<String, dynamic> enrollParamsJson = enrollParams.toJson();
+    enrollParamsJson.removeWhere(_removeElements);
+    if (enrollParamsJson.isNotEmpty) {
+      sb.write(':${jsonEncode(enrollParamsJson)}');
     }
     sb.write('\n');
     return sb.toString();
   }
 
+  /// Checks if the key and its value should be added to the enroll verb command.
```

Comment:
method naming(remove) and this dart doc comment(add) is contradictory. change the dart doc comment

Reply:
change made as part of [feddc9e](https://github.com/atsign-foundation/at_libraries/pull/530/commits/feddc9e80d5327223fa0a5fed5f6ff9a3173fe56)